app/core/utils.coffee

```diff
@@ -119,6 +119,11 @@ courseIDs =
   COMPUTER_SCIENCE_5: '569ed916efa72b0ced971447'
   COMPUTER_SCIENCE_6: '5817d673e85d1220db624ca4'
 
+# TODO add when final courses content created for voyager 
+voyagerCourseIDs = [
+  ''
```

Comment:
Could this empty string create issues, and could an empty array work?

Reply:
Yeah I think an empty array would work, will update. Thank you